source-code/core2/src/fullExample.ts

```diff
@@ -70,15 +71,10 @@ const exampleInlangConfig: InlangConfig = {
   }
 }
 
-// 1. The JSON config needs to be read manually.  
-//    - an app instance can't know where a config is located
-//    - gives apps more freedom to load different configs if desired (testing, etc.)
-const config = fs.readFile("./inlang.config.json")
-
-// 2. Create the app instance
+// 1. Create the app instance
 //    - env needs be created
 const inlang = createApp({
-	config,
+	configPath: "/hello/inlang.config.json",
```

Comment:
we could encourage people to pass the config path via an env variable, but certainly not must because we do not have sensitive information (yet) in the config (google translate keys)
furthermore, do we want to allow URLs here? (makes read/write hard or impossible)

https://www.prisma.io/docs/reference/database-reference/connection-urls#env

Reply:
> we could encourage people to pass the config path via an env variable, but certainly not a must because we do not have sensitive information (yet) in the config (google translate keys)

Yeah... no use case. 

> furthermore, do we want to allow URLs here? (makes read/write hard or impossible)

Prob not. In the future we can add a new prop `config` that allows manual passing of the config. 

Please resolve this conversation if you agree that we shouldn't change anything here.

Reply:
Url to the inlang.config.json will never be sensitive right? 

> https://www.prisma.io/docs/reference/database-reference/connection-urls#env

Haha slowly this really feels like building git as a database :DDD

Reply:
> Url to the inlang.config.json will never be sensitive right? 

No, it won't. I am closing this discussion. 

> Haha slowly this really feels like building git as a database :DDD

Right now, we are talking about building lisa prisma (the thing in memory) :)